edsnlp/pipelines/misc/sections/sections.py

```diff
@@ -79,9 +79,12 @@ def __init__(
         self.add_patterns = add_patterns
         if add_patterns:
             for k, v in sections.items():
-                sections[k] = [
-                    r"(?<=\n[^\n]{0,5})" + ent + r"(?=[^\n]{0,5}\n)" for ent in v
-                ]
+                if k != "introduction_CR":
+                    sections[k] = [
+                        r"(?<=\n[^\n]{0,5})" + ent + r"(?=[^\n]{0,5}\n)" for ent in v
+                    ]
+                else:
+                    sections[k] = [ent + r"(?=[^\n]{0,5}\n)" for ent in v]
```

Comment:
Great idea, but after testing I would suggest a slight modification of the new regex
```suggestion
                sections[k] = [
                    r"(?<=(?:\n|^)[^\n]{0,5})" + ent + r"(?=[^\n]{0,5}\n)" for ent in v
                ]
```